node/pkg/governor/governor_monitoring.go

```diff
@@ -600,7 +601,7 @@ func (gov *ChainGovernor) publishConfig(hb *gossipv1.Heartbeat, sendC chan<- []b
 
 	digest := ethCrypto.Keccak256Hash(append(governorMessagePrefixConfig, b...))
 
-	sig, err := guardianSigner.Sign(digest.Bytes())
+	sig, err := guardianSigner.Sign(context.Background(), digest.Bytes())
```

Comment:
There's isn't a Context being passed by the `cmd/guardiand/node.go`?

Reply:
The run function in `node/pkg/p2p/p2p.go` that gets produced does accept a context. I've modified `CollectMetrics` in governor monitoring to accept the context, and pass it to `publishConfig`, so it gets passed to `Sign`.